action.yaml

```diff
@@ -20,9 +20,15 @@ runs:
         then
```

Comment:
If we're going straight into master I think we should also increase the version number. Here that is just updating the version mentioned in the documentation